bakery/src/pipelines/distribution.js

```diff
@@ -86,7 +86,8 @@ const pipeline = (env) => {
       taskValidateXhtml({
         image: { tag: lockedTag },
         inputSource: 'jsonified-book',
-        inputPath: 'jsonified/*@*.xhtml'
+        inputPath: 'jsonified/*@*.xhtml',
+        validationName: 'duplicate-id'
```

Comment:
This isn't directly related to this change, but it occurred to me when we eventually turn on all the checks the `validationName` would be an empty string with the current implementation of `xhtml-validator`. Should we create an issue to make a change there so it's `all` or something more explicit for later?

Reply:
Well, some validations are subsets of others. If duplicate-id passes, link-to-duplicate-id must also pass. I think it would be handy to take an argument list of the validations to run. (But I agree with 'all', if we don't really care about doing a little bit of extra work twice, which I don't really, but a list may more flexible)